### DISCUSSION TITLE
Bagaimana treatment data time series yang sudah ditransform dengan Box Cox?

### DISCUSSION THREAD

**Discussion by Hertzspr** [teamalgoritma/community#164] [Q&A] [answered]
Untuk mengubah data time series menjadi lebih stabil dan stasiuner, saya terapkan Box Cox transformation dengan nilai lambda yang dihitung dari fungsi guerrero sebagai berikut. 

Sebelum
![1](https://user-images.githubusercontent.com/95233522/176501687-16e5a3b4-772a-4277-a0d7-5375e51b39f8.JPG)
Sesudah
![2](https://user-images.githubusercontent.com/95233522/176501700-aa71b94b-0493-4c2f-ae65-672ddbca0cd2.JPG)

Pertanyaanya: setelah proses modeling dan forecast nanti, apakah hasil forecastnya perlu dilakukan transformasi Box Cox kembali? Dengan nilai negatif dari lambda yang digunakan di awal?

---
catatan: saya menggunakan pendekatan tidyverts: tsibble, fable, dkk.

**Comment by Hertzspr** [comment]
Maaf, saya belum yakin apa ini jawabannya, tapi dari [link ini](https://otexts.com/fpp3/ftransformations.html)

Katanya kalau pake fable, forecastingnya otomatis sudah dibacktransformed kalau di sejarah modelnya pernah ada transformasi. Ini benar nggak ya?
![answer](https://user-images.githubusercontent.com/95233522/176740884-a64246ed-6e66-4b71-88b7-f171e98785b0.JPG)

**Comment by triarm** [comment, marked as answer]
Hi @Hertzspr ,
Pada umumnya ketika kita melakukan transformasi data saat akan melakukan pemodelan maka kita perlu transformasi kembali ke nilai awal saat akan melakukan forecasting agar nilai dan satuannya sesuai dengan keadaan sebenarnya. Untuk kasus transformasi Box-Cox dengan package `fable`, bapak dapat memanfaatkan fungsi `box_cox()` untuk melakukan transformasi Box-Cox dan fungsi `inv_box_cox()` untuk melakukan reverse transformation (transformasi ke nilai awal). Berikut adalah [dokumentasi](https://fabletools.tidyverts.org/reference/box_cox.html) terkait transformasi tersebut.

**Reply by Hertzspr** [reply]
Terima kasih.
